RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate AuthenticationWebView to Typescript

## Proposed changes


## Issue(s)	


## How to test or reproduce

- You need to connect to any workspace, you could test with the `open.rocket.chat`
- Then you must click on any oAuth authentication, as: `github`, `gitlab`... except the `goole oAuth`
- And should be open a web view in-app to do the authentication

## Screenshots

## Types of changes



- [] Bugfix (non-breaking change which fixes an issue)

## Checklist



## Further comments
